includes/paypro/wc/woocommerce.php

```diff
@@ -66,11 +66,11 @@ public function cancelOrder(WC_Order $order, $payment_hash)
         $message = sprintf(__('PayPro payment cancelled (%s)', 'paypro-gateways-woocommerce'), $payment_hash);
 
         if(PayPro_WC_Plugin::$settings->automaticCancellation())
-            $order->cancel_order($message);
+            $order->update_status('cancelled', $message);
         else
             $order->add_order_note($message);
 
-        $this->removeOrderPaymentHash($order);
+        $this->removeOrderPaymentHashes($order);
```

Comment:
Was this a bug too?

Reply:
Yes, the error was thrown after the order cancelling.